.clang-format

```diff
@@ -1,62 +1,74 @@
-﻿BasedOnStyle: WebKit
+---
+BasedOnStyle: WebKit
 
 AccessModifierOffset: -4
-AlignEscapedNewlinesLeft: true
+AlignAfterOpenBracket: Align
+AlignConsecutiveAssignments: false
+AlignConsecutiveDeclarations: false
+AlignEscapedNewlines: Left
+AlignOperands: true
 AlignTrailingComments: true
-AllowAllParametersOfDeclarationOnNextLine: false
+AllowAllParametersOfDeclarationOnNextLine: true
 AllowShortBlocksOnASingleLine: false
 AllowShortCaseLabelsOnASingleLine: false
 AllowShortFunctionsOnASingleLine: None
-AllowShortIfStatementsOnASingleLine: true
+AllowShortIfStatementsOnASingleLine: false
 AllowShortLoopsOnASingleLine: false
-AlwaysBreakAfterDefinitionReturnType: false
-AlwaysBreakBeforeMultilineStrings: true
+AlwaysBreakAfterDefinitionReturnType: None
+AlwaysBreakAfterReturnType: None
+AlwaysBreakBeforeMultilineStrings: false
 AlwaysBreakTemplateDeclarations: true
-
 BinPackArguments: false
 BinPackParameters: true
-BreakBeforeBinaryOperators: NonAssignment
+BreakBeforeBinaryOperators: None
 BreakBeforeBraces: Allman
-BreakBeforeTernaryOperators: true
-BreakConstructorInitializersBeforeComma: true
-
+BreakBeforeTernaryOperators: false
+BreakConstructorInitializers: AfterColon
```

Comment:
Ah ok i see.

Reply:
To command it out wont help here i think, because clang-format will fall back to the default defined by WebKit. And after all i searched many files for an example to this and i found this:
```
MetadataEntry::MetadataEntry()
 : valid(false),
   zoom(1),
   page(0),
   time(0)
{
}
```

This looks like should stick to AfterColon

Reply:
That's what we're looking for.. but so far the only way I've gotten close to it is with the unlikely combination ( that still gets the commas in the wrong place) : 
```
ConstructorInitializerAllOnOneLineOrOnePerLine: false
###BreakConstructorInitializers: BeforeColon
```
 clang-format is still a little flaky and it looks like it requires some trial-and-error.

Reply:
Left is current format, middle is current clang-format, right has BreakConstructorInitializers commented out and ConstructorInitializerIndentWidth: 1
![image](https://user-images.githubusercontent.com/2329541/59894093-82ca7b80-939c-11e9-970d-49548c0bf91f.png)